pandas/core/apply.py

```diff
@@ -293,20 +291,9 @@ def apply_series_generator(self):
                 res_index = res_index.take(successes)
 
         else:
-            try:
-                for i, v in enumerate(series_gen):
-                    results[i] = self.f(v)
-                    keys.append(v.name)
-            except Exception as err:
-                if hasattr(err, "args"):
-
-                    # make sure i is defined
-                    if i is not None:
-                        k = res_index[i]
-                        err.args = err.args + (
-                            "occurred at index %s" % pprint_thing(k),
-                        )
-                raise
+            for i, v in enumerate(series_gen):
+                results[i] = self.f(v)
+                keys.append(v.name)
```

Comment:
I personally think this can be informative, though. 
Is this that problematic? (we are raising the same exception again, so it is not hiding any exceptions?)

Reply:
The index information can be useful, but this can also make it harder to reason about what parts of the code are exception-raising.

This can go either way.  If there's a consensus to keep it, I'll revert.